### DISCUSSION TITLE
Configuration of defaults e.g. button size

### DISCUSSION THREAD

**Discussion by maku** [themesberg/flowbite-svelte#285] [Q&A]
Would it be possible to define a default configuration (e.g. all Buttons have size="xs")

**Comment by shinokada** [comment]
As you know the `size` prop is exported as below. 

```
  export let size: 'xs' | 'sm' | 'md' | 'lg' | 'xl' = group ? 'sm' : 'md';
```

What I can think of is that you can copy/paste the Button component to your lib dir and change the default.

**Comment by maku** [comment]
@shinokada Ok I understand that, In my naivety I thought that this could possibly be configured via the plugin. 

But, when I think about it it would be a nice feature to be able to config that :-)
